### PR TITLE
Rebase #76 and add tests and change log

### PR DESCRIPTION
**Pull request by oschwald**
Closes #76.

### CODE REVIEW

File: t/notes.t [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+use utf8;
+
+use lib 't/lib';
+
+use Apache::Test qw(-withtestmore);
+use Apache::TestRequest;
+use JSON::XS qw( decode_json );
+use Test::ModMaxMindDB;
+
+my $url = '/cgi-bin/valid-db/json-env';
```
Comment (horgh) [inline t/notes.t]: Should this be hitting /notes? I don't see us actually hitting the new path, nor checking what the test package is doing.

File: src/mod_maxminddb.c [hunk 2]
```diff
@@ -153,6 +166,16 @@ module AP_MODULE_DECLARE_DATA maxminddb_module = {
     maxminddb_register_hooks /* register hooks                      */
 };
 
+static void maxminddb_kv_set(maxminddb_config *conf,
```
Comment (horgh) [inline src/mod_maxminddb.c]: There's a call to apr_table_set() in set_network_environment_variable() - should we use this function there too?

File: t/notes.t (hunk 1 above)
Comment (oschwald) [reply t/notes.t]: That explains why it surprisingly passed right away. :rofl:

File: src/mod_maxminddb.c (hunk 2 above)
Comment (oschwald) [reply src/mod_maxminddb.c]: Good call. I missed that on rebase.

**Pull request merged**